[Plugin] Add `codewithdennis/filament-advanced-components` plugin

CodeWithDennis opened the pull request:
This is a paid plugin. As instructed, I have invited Dan to the private repository.

saade reviewed (approved):
Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!

Please let us know when you've addressed these issues.

saade commented:
@danharrin lmk when i can merge this

danharrin commented:
@saade all good

CodeWithDennis commented:
> Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!
> 
> Please let us know when you've addressed these issues.

Done =)
